src/main/resources/db/migration/v0.27.0.1__add_constructs.sql

```diff
@@ -0,0 +1,144 @@
+	id bigint PRIMARY KEY,
+	singleconstruct_id bigint,
+	componentsymbol varchar(255),
+	taxon_curie varchar(255),
+	taxontext varchar(255)
+);
+
+ALTER TABLE constructcomponentslotannotation ADD CONSTRAINT constructcomponentslotannotation_id_fk FOREIGN KEY (id) REFERENCES slotannotation (id);
+ALTER TABLE constructcomponentslotannotation ADD CONSTRAINT constructcomponentslotannotation_singleconstruct_id_fk FOREIGN KEY (singleconstruct_id) REFERENCES construct (id);
+ALTER TABLE constructcomponentslotannotation ADD CONSTRAINT constructcomponentslotannotation_taxon_curie_fk FOREIGN KEY (taxon_curie) REFERENCES ncbitaxonterm (curie);
+
+CREATE INDEX constructcomponentslotannotation_singleconstruct_index ON constructcomponentslotannotation USING btree (singleconstruct_id);
+CREATE INDEX constructcomponentslotannotation_componentsymbol_index ON constructcomponentslotannotation USING btree (componentsymbol);
+CREATE INDEX constructcomponentslotannotation_taxon_index ON constructcomponentslotannotation USING btree (taxon_curie);
+
+CREATE TABLE constructcomponentslotannotation_aud (
+    id bigint NOT NULL,
+    rev integer NOT NULL,
+	singleconstruct_id bigint,
+	componentsymbol varchar(255),
+	taxon_curie varchar(255),
+	taxontext varchar(255)
+);
+
+ALTER TABLE constructcomponentslotannotation_aud ADD PRIMARY KEY (id, rev);
+
+ALTER TABLE constructcomponentslotannotation_aud ADD CONSTRAINT constructcomponentslotannotation_aud_id_rev_fk FOREIGN KEY (id, rev) REFERENCES slotannotation_aud (id, rev);
+	
+	
+CREATE TABLE constructcomponentslotannotation_note (
+	constructcomponentslotannotation_id bigint,
+	relatednotes_id bigint
+);
+	
+ALTER TABLE constructcomponentslotannotation_note ADD CONSTRAINT constructcomponentslotannotation_note_ccsa_id_fk FOREIGN KEY (constructcomponentslotannotation_id) REFERENCES constructcomponentslotannotation (id);
+ALTER TABLE constructcomponentslotannotation_note ADD CONSTRAINT constructcomponentslotannotation_note_relatednotes_id_fk FOREIGN KEY (relatednotes_id) REFERENCES note (id);
+
+CREATE INDEX constructcomponentsa_note_ccsa_id_index ON constructcomponentslotannotation_note USING btree (constructcomponentslotannotation_id);
+CREATE INDEX constructcomponentsa_note_relatednotes_id_index ON constructcomponentslotannotation_note USING btree (relatednotes_id);
+
+CREATE TABLE constructcomponentslotannotation_note_aud (
+	constructcomponentslotannotation_id bigint,
+	relatednotes_id bigint,
+	rev integer NOT NULL,
+	revtype smallint
+);	
+	
+ALTER TABLE constructcomponentslotannotation_note_aud ADD PRIMARY KEY (constructcomponentslotannotation_id, relatednotes_id, rev);
+
+ALTER TABLE constructcomponentslotannotation_note_aud ADD CONSTRAINT constructcomponentslotannotation_note_aud_rev_fk FOREIGN KEY (rev) REFERENCES revinfo (rev);
+
+	
+INSERT INTO vocabulary (id, name, vocabularylabel) VALUES (nextval('hibernate_sequence'), 'Construct Component Note Type', 'construct_component_note_type');
```

Comment:
After the quarkus 3.0 upgrade `nextval('hibernate_sequence')` will need to be changed to the specific sequence for the table.

Reply:
Yup - we have a lot of these in the migrations